browser_install.py

```diff
@@ -195,9 +195,9 @@ def firefox(self, channel):
                 # Create an ini file for the installer to use
                 ini_file = os.path.join(self.dir, 'firefox.ini')
                 with open(ini_file, 'wb') as ini:
-                    ini.write("[Install]\n")
-                    ini.write("InstallDirectoryName={0}\n".format(channel))
-                    ini.write("MaintenanceService=false\n")
+                    ini.write(bytes('[Install]\n', 'utf-8'))
+                    ini.write(bytes('InstallDirectoryName={0}\n'.format(channel), 'utf-8'))
+                    ini.write(bytes('MaintenanceService=false\n', 'utf-8'))

```

Comment:
Dis this part also work in python 2.7?  I'll see about testing but I assume opening the file in text mode instead might be more compatible.

Reply:
I changed it to 'w' and now it supports 2.7. Verified on another windows machine running under Python 2.7, and verify it also works under Python 3.9